touchforms/spec/javascripts/entrycontrols_full_spec.js

```diff
@@ -121,17 +121,22 @@ describe('Entries', function() {
 
     it('Should return DateEntry', function() {
         questionJSON.datatype = Formplayer.Const.DATE;
-        questionJSON.answer = '90-09-26';
+        questionJSON.answer = '1990-09-26';
 
         entry = (new Question(questionJSON)).entry;
         expect(entry instanceof DateEntry).toBe(true);
         expect(entry.templateType).toBe('date');
 
-        entry.answer('87-11-19');
+        entry.answer('1987-11-19');
```

Comment:
so this test doesn't actually test the parsing of the date. it all happens in `$datepicker` when the user selects a date. the only parsing that happens is after the first render which is hard to test in a js test. the `DateEntry` doesn't do any validation of the date format because i decided that was the `$datepicker`'s job. i only changed this so that if people look at the tests for an example, it shows you what an actual answer would look like. this test just ensures that you can instantiate a `DateEntry` object